resources/js/pages/social/components/ManageApplicationsModal.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+    <Loading v-if="loading" />
+    <div v-else>
+        <template v-for="application in applications" :key="application.id">
+            <p>
+                {{`${application.username}, applied on: ${application.applied_at}`}}
+                <button class="m-1" @click="rejectApplication(application)">
+                    {{ $t('reject-group-application') }}
+                </button>
+                <button class="m-1" @click="acceptApplication(application)">
+                    {{ $t('accept-group-application') }}
+                </button>
+            </p>
+        </template>
+    </div>
+</template>
+
+<script setup>
+import {useGroupStore} from '/js/store/groupStore';
+import {ref, onMounted, computed} from 'vue';
+const groupStore = useGroupStore();
+
+const emit = defineEmits(['reloadGroups']);
+const loading = ref(true);
+onMounted(() => {
+    load();
+});
+async function load() {
+    await groupStore.fetchApplications(props.group);
+    loading.value = false;
+}
+
+const applications = computed(() => groupStore.applications);
```

Comment:
I've made some changes to how often the store is used, when fetching applications for a single group, it doesn't need to be stored. Instead you can just turn this into a ref(null), return the data from the store, and then setting it in the load function. The examples are in another branch apparently.
Either way, in load: `applications.value = await groupStore...`